frontend/src/metabase/dashboard/components/DashCard/DashCardParameterMapper/DashCardCardParameterMapper.unit.spec.jsx

```diff
@@ -37,7 +37,7 @@ const setup = options => {
       card={createMockCard()}
       dashcard={createMockDashboardCard()}
       editingParameter={{}}
-      target={null}
+      target={undefined}
```

Comment:
if you pass nothing, it's going to be undefined

Reply:
I sad this pattern in a few other places in our codebase so I thought maybe we had some internal convention to explicitly mark some props as undefined in tests?

For example
https://github.com/metabase/metabase/blob/master/frontend/src/metabase/query_builder/components/expressions/ExpressionWidget.unit.spec.tsx#L196

Reply:
> @kamilmielnik do we have a pattern?

I don't think so.